tests/Yaapii.Atoms.Tests/Text/EndsWithTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Xunit;
+using Yaapii.Atoms.Text;
+
+namespace Yaapii.Atoms.Tests.Text
+{
+    public sealed class EndsWithTests
+    {
+        [Fact]
+        public void FindsTextEnd()
+        {
+            var x =
+                new EndsWith(
+                    new TextOf("Im an text with a realy good end!"),
+                    new TextOf("od end!")
+                );
+            Assert.True(x.Value());
+        }
+
+        [Fact]
+        public void TakesTailAsString()
+        {
+            var x =
+                new EndsWith(
+                    new TextOf("Im an text with a realy good end!"),
+                    "od end!"
+                );
+            Assert.True(x.Value());
+        }
+
+        [Fact]
+        public void FailsOnWrongTail()
+        {
+            var x =
+                new EndsWith(
+                    new TextOf("Im an text with a realy good end!"),
+                    new TextOf("od end")
+                );
+            Assert.False(x.Value());
+        }
+
+        [Fact]
+        public void FailsOnCaseMismatch()
+        {
+            var x =
+                new EndsWith(
+                    new TextOf("Im an text with a realy good end!"),
+                    new TextOf("od enD!")
+                );
+            Assert.False(x.Value());
+        }
+
+        [Fact]
+        public void FindsMultilineTextEnd()
```

Comment:
This is a unneeded test. Why should this ever fail? This could only fail when a string concatenation function would not work and this is a completely different function.

please think a bit more if a test is really necessary.